python_modules/libraries/dagster-dbt/dagster_dbt/cloud_v2/resources.py

```diff
@@ -99,6 +100,44 @@ def unique_id(self) -> str:
         """
         return f"{self.project_id}-{self.environment_id}"
 
+    @cached_property
+    def account_name(self) -> Optional[str]:
```

Comment:
hm but it's optional?

Reply:
It's optional because of the dbt Cloud API, which doesn't mark the field as required, see [here](https://docs.getdbt.com/dbt-cloud/api-v2#/operations/Retrieve%20Account). Same thing for projects and environments.

Reply:
We could return a name like `f"account_{account_id}"` if the API response is `None` though